Microsoft.Toolkit.Forms.UI.XamlHost/Microsoft.Toolkit.Forms.UI.XamlHost.csproj

```diff
@@ -1,20 +1,17 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
 
   <PropertyGroup>
-    <TargetFramework>net462</TargetFramework>
+    <TargetFrameworks>net462;netcoreapp3.0</TargetFrameworks>
     <RootNamespace>Microsoft.Toolkit.Forms.UI.XamlHost</RootNamespace>
     <AssemblyName>Microsoft.Toolkit.Forms.UI.XamlHost</AssemblyName>
-    <RunCodeAnalysis>true</RunCodeAnalysis>
 
     <Title>Windows Community Toolkit XAMLHost for Windows Forms</Title>
     <Description>This library provides XAML islands interop helpers for Windows Forms. It is part of the Windows Community Toolkit.</Description>
     <PackageTags>XAML Islands Win32 Forms WindowsForms WinForms XamlHost</PackageTags>
+  </PropertyGroup>
 
-    <!--
-    Needs the XamlRoot API, which is available in Insiders SDK 18298 on 19-Dec 2018
-    https://blogs.windows.com/buildingapps/2018/12/19/windows-10-sdk-preview-build-18298-available-now
-    -->
-    <TargetPlatformMinVersion>10.0.18298.0</TargetPlatformMinVersion>
```

Comment:
This is the correct place to override `TargetPlatformMinVersion` rather than DirectoryInfo.props/.targets

Reply:
Since the Webview projs are the only ones that uses a different version, I inverted that. Webview is the only one that overrides it. Wouldn't that work?